docs/orchestration/ui/interactive-api.md

```diff
@@ -93,7 +93,7 @@ query {
     }
 }
 ```
-You can see more about query filters in the [Hasura Docs](https://hasura.io/docs/1.0/graphql/manual/queries/query-filters.html).
+To learn more about the various query filters available (there are many more than the examples above!) head over to the [Hasura Docs](https://hasura.io/docs/2.0/graphql/core/databases/postgres/queries/index.html#queries).
```

Comment:
We're using Hasura 1.x so we should probably link to https://hasura.io/docs/1.0/graphql/core/queries/index.html -- I don't think there's much a difference yet but I don't want people to be confused when Hasura 2.x examples don't work.